src/component/contents/DraftEditorBlocks.react.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
```

Comment:
You are right I think we should keep on with the current approach. We can always revisit it later if needed.